components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationMgtValidator.java

```diff
@@ -82,7 +75,6 @@ public boolean doPreCreateApplication(ServiceProvider serviceProvider, String te
         return true;
     }
 
-    @Override
     public boolean doPreUpdateApplication(ServiceProvider serviceProvider, String tenantDomain,
```

Comment:
Since this is not a listener, we can change the methods with appropriate names (i.e. pre and post prefixes are not needed)